gpjax/kernels/base.py

```diff
@@ -0,0 +1,55 @@
+import jax.numpy as jnp
+from objax import Module
+from typing import Callable, Optional
+from jax import vmap
+from ..parameters import Parameter
+
+
+class Kernel(Module):
+    """
+    Base class for all kernel functions. By inheriting the `Module` class from Objax, seamless interaction with model parameters is provided.
+    """
+    def __init__(self, name: Optional[str] = None):
+        """
+        Args:
+            name: Optional naming of the kernel.
+        """
+        self.name = name
+        self.spectral = False
+
```

Comment:
It's good that we've removed the `Stationary` inheritance, but it may be beneficial to have stationarity as a property of the kernel class.